### PR TITLE
Update vagrant

### PR DESCRIPTION
**Pull request by kimpepper**
* Updates vagrant to the pnx/lamp box
* Includes php 5.5

Steps to upgrade:

* vagrant destroy -f
* rm -rf .vagrant
* vagrant up

All commands should be run inside the vm

* vagrant ssh
* composer install --prefer-dist
* Remove sites/default/settings.php
* phing reinstall

### CODE REVIEW

**Comment by jibran** [comment]
Can we enable apc and uploadporgess in the lamp box?

**Comment by kimpepper** [comment]
APCu ? Php 5.5 doesn't need APC, it has it's own opcode cache.

On 1 May 2015 at 12:23, Jibran Ijaz <notifications@github.com> wrote:

> Can we enable apc and uploadporgess in the lamp box?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nickschuch/vd8/pull/26#issuecomment-98028182>.
>



-- 
Kim Pepper
Technical Director
PreviousNext

desk: 02 8005 8670
mobile: 0414 212 530
skype: kim.pepper

**Comment by nickschuch** [comment]
@jibran are you ok if I merge this? I just want to make sure it won't break workflow.

**Comment by jibran** [comment]
Let me test it first.

File: composer.json [hunk 1]
```diff
@@ -1,6 +1,9 @@
 {
+  "name": "pnx/vd8",
```
Comment (jibran) [inline composer.json]: Ahm

**Comment by jibran** [comment]
This works great. +1 to merge.

File: composer.json (hunk 1 above)
Comment (nickschuch) [inline composer.json]: Ahm is correct :)

**Comment by theodorosploumis** [comment]
Are there any changes need to be done on Docs about this merge? 
Currently `vagrant up` stops on step:

```==> lamp: Mounting NFS shared folders ```

**Pull request merged**